boards/raspberrypi/rpi_pico/rpi_pico_rp2040_w.dts

```diff
@@ -1,9 +1,67 @@
 /*
  * Copyright (c) 2023 Dave Rensberger - Beechwoods Software
+ * Copyright (c) 2024 Steve Boylan <stephen.boylan@beechwoods.com>
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 /dts-v1/;
 
 #include "rpi_pico-common.dtsi"
+
+&pinctrl {
+	pio0_spi0_default: pio0_spi0_default {
+		/* gpio 25 is used for chip select, not assigned to the PIO */
+		group1 {
+			pinmux = <PIO0_P29>;
+		};
+	};
+
+	airoc_wifi_default: airoc_wifi_default {
+		/* Control of GPIO24 is done through the WiFi driver */
+		group1 {
+			pinmux = <PIO0_P24>;
+			input-enable;
+		};
+	};
+
+	airoc_wifi_host_wake: airoc_wifi_host_wake {
+		/* Assign GPIO24 to SIO (GPIO) for use as an interrupt source */
+		group1 {
+			/* Lacking a specific SIO pin definition, use the RP2040_PINMUX macro */
+			pinmux = <RP2040_PINMUX(24, RP2_PINCTRL_GPIO_FUNC_SIO)>;
+			input-enable;
+		};
+	};
+};
+
+&pio0 {
+	status = "okay";
+
+	pio0_spi0: pio0_spi0 {
```

Comment:
Very minor nit/suggestion (should not be a blocker): Could/should the device tree order of properties match the Linux DTS coding style? https://docs.kernel.org/devicetree/bindings/dts-coding-style.html#order-of-properties-in-device-node

Reply:
That's a good suggestion.  If it becomes necessary to make any required changes for this pull request, I'll do that.